pkg/kubectl/cmd/version.go

```diff
@@ -17,16 +17,24 @@ limitations under the License.
 package cmd
 
 import (
+	"errors"
 	"fmt"
-	"io"
-
 	"github.com/spf13/cobra"
+	"io"
 
+	"encoding/json"
+	"github.com/ghodss/yaml"
+	apimachineryversion "k8s.io/apimachinery/pkg/version"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/version"
 )
 
+type VersionObj struct {
```

Comment:
Please just call it Version. I haven't seen Obj as a suffix in the rest of the codebase.